saleor/order/utils.py

```diff
@@ -790,6 +791,25 @@ def get_voucher_discount_for_order(order: Order) -> Money:
     raise NotImplementedError("Unknown discount type")
 
 
+def get_manual_discounts_for_order(order: Order) -> Money:
+    """Calculate discount value depending on discount types.
+    """
+    order_discounts = OrderDiscount.objects.filter(order_id=order, type="manual")
+    return order_discounts
+
+
+def get_order_discount_position(name: str, value: Union[Decimal, Decimal]):
```

Comment:
value type should be just Decimal, Union is for different data types which is not the case here